plugins/alignments/src/SNPCoverageAdapter/SNPCoverageAdapter.ts

```diff
@@ -35,29 +35,28 @@ export default (pluginManager: PluginManager) => {
   const MyConfigSchema = MyConfigSchemaF(pluginManager)
 
   function generateInfoList(table: NestedFrequencyTable) {
-    const infoList = []
     const overallScore = table.total()
 
     // log info w/ base name, total score, and strand breakdown
-    // eslint-disable-next-line guard-for-in
-    for (const base in table.categories) {
-      const strands = table.categories[base].categories as {
+    const infoList = Object.entries(table.categories).map(([base, strand]) => {
+      const strands = strand.categories as {
         [key: string]: number
       }
       const score = Object.values(strands).reduce((a, b) => a + b, 0)
-      infoList.push({
+      return {
         base,
         score,
         strands,
-      })
-    }
+      }
+    })
 
     // sort so higher scores get drawn last, reference always first
     infoList.sort((a, b) =>
       a.score < b.score || b.base === 'reference' ? 1 : -1,
     )
 
     // add overall total to end
+    // @ts-ignore
```

Comment:
what is motivating this ts-ignore?

Reply:
the structure of infoList assumed strand existed in all cases. I refactored now to avoid it